assembl/static2/js/app/components/debate/tableOfIdeas/ideasLevelMobile.jsx

```diff
@@ -0,0 +1,90 @@
+// @flow
+import React from 'react';
+import truncate from 'lodash/truncate';
+import { Col } from 'react-bootstrap';
+import classnames from 'classnames';
+import IdeaPreview from './ideaPreview';
+import { get as getRoute } from '../../../utils/routeMap';
+import { getDiscussionSlug } from '../../../utils/globalFunctions';
+
+type Props = {
+  ideas: Array<Idea>,
+  identifier: string,
+  ideaLevel: number,
+  nbLevel: number,
+  selectedIdeasId: string,
+  setSelectedIdeas: Function
+};
+
+const IdeasLevelMobile = ({ ideas, identifier, ideaLevel, nbLevel, selectedIdeasId, setSelectedIdeas }: Props) => {
+  const slug = getDiscussionSlug();
+  const stringMaxLength = (level) => {
+    switch (level) {
+    case 1:
+      return 80;
+    case 2:
+      return 50;
+    case 3:
+      return 30;
+    default:
+      return 30;
+    }
+  };
+  return (
+    <div
+      className={classnames('slider-container', 'mobile-slider-container', {
+        'thumbnails-mobile-slider': nbLevel > 1 && ideaLevel < nbLevel
+      })}
+      style={nbLevel > 1 && ideaLevel < nbLevel ? { height: '160px' } : null}
+    >
+      <div
+        className={classnames('slider', { 'thumbnails-slider': nbLevel > 1 && ideaLevel < nbLevel })}
+        style={
+          nbLevel > 1 && ideaLevel < nbLevel
```

Comment:
ideaLevel can only be < nbLevel if nbLevel > 1 right ? so you don't need to check for nbLevel > 1 I think

Reply:
Also, this check is done 3 times so maybe you can put it in a variable

Reply:
yes good catch